client/components/download-link/renderers/docx-renderer.js

```diff
@@ -493,9 +493,15 @@ export default (application, sections, values, updateImageDimensions) => {
   };
 
   const renderDuration = (doc, value) => {
-    let years = value.years === 1 ? 'Year' : 'Years';
-    let months = value.months === 1 ? 'Month' : 'Months';
-    doc.createParagraph(`${value.years} ${years} ${value.months} ${months}`).style('body');
+    let months = 0;
+    let years = 5;
+    if (value) {
+      months = value.months;
+      years = value.years;
```

Comment:
```suggestion
      years = value.years || years;
```

Reply:
These two changes because `value` could be of the form `{ months: 0 }` so is truthy but doesn't contain all values.